### ISSUE TITLE
Additional ingress annotations required

### ISSUE THREAD

**Issue by foggy-glasses** [ContainerSolutions/trow#83]
While attempting to push an image that was around 200 MB in size for the very first time since the registry was setup, I ran into Error 413 - Request Entity too large.

I then increased the client_max_body_size, proxy-body-size and client-body-size in my ingress to "500m" upon which it worked. 

I believe the documentation should also mention adding and adjusting these annotations in the ingress used.

**Comment by amouat** [comment]
Thanks, that's a good suggestion. Which ingress are you using?

**Comment by amouat** [comment]
Thanks, that's a good suggestion.

I actually played with this already for one of the kustomize targets: https://github.com/ContainerSolutions/trow/blob/master/install/overlays/cert-manager-nginx/ingress.yaml

I *think* you should be ok to only set `proxy-body-size` (https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#custom-max-body-size). Could you test that and let me know if it works with only that change?

**Comment by foggy-glasses** [comment]
@amouat I just confirmed it. It works only with setting proxy-body-size as an annotation in the Ingress to "0". 

I do have another query if that's alright.. I tagged an image as myregistry-domain.com/platform/service/one:03_2020. When I attempt to push this to myregistry-domain.com, I get the error 
`error parsing HTTP 404 response body: json: cannot unmarshal string into Go value of type struct { Errors []errcode.Error }: "\"404 page not found\""`

The trow-pod logs show this
`_[ERROR] No matching routes for POST /v2/platform/service/one/blobs/uploads/?from=one&mount=sha256%3Af5d23c7fed465a9eb762fc4c3cccd551a05914aba42492ceb972497db4df38bf.`

Not sure if I'm missing something or if this is totally incorrect. I have an EKS cluster running off ECR platform where I tag images as such and it gets pushed without any issues. Of course, I do first create a repository in ECR with the same format before pushing in.
